package.json

```diff
@@ -114,7 +117,10 @@
   "homepage": "https://github.com/ckeditor/ckeditor5-react",
   "files": [
     "dist",
-    "scripts/postinstall.js"
+    "scripts/postinstall.js",
+    "README.md",
+    "CHANGELOG.md",
```

Comment:
Previously, we did not publish the `CHANGELOG.md` - see https://www.npmjs.com/package/@ckeditor/ckeditor5-react?activeTab=code. Is it needed on npm?